BDArmory/Control/BDModulePilotAI.cs

```diff
@@ -948,10 +957,7 @@ void FlyToPosition(FlightCtrlState s, Vector3 targetPosition)
 			targetPosition = vessel.transform.position + (currTargetDir * 100);
 
 
-			if(BDArmorySettings.DRAW_DEBUG_LINES)
-			{
-				flyingToPosition = targetPosition;
-			}
+			flyingToPosition = targetPosition;
```

Comment:
Collision detection use this (line 655), so assigning to it all the time, not only with debug lines on.